docs/vercel.json

```diff
@@ -101,7 +100,7 @@
     {
       "source": "/",
       "has": [{ "type": "host", "value": "discord.authjs.dev" }],
-      "destination": "https://discord.gg/kuv7wXkHY4"
```

Comment:
I had just updated this to a no-timeout link a week or two ago. Are sure that link wasn't working? 🤔 

Otherwise, LGTM!

Reply:
I'm sorry. I included unintended changes.
It has been corrected to the correct one.